README.md

```diff
@@ -41,8 +41,9 @@ Init client for API services. Get APIKey/SecretKey from your binance account.
 var (
     apiKey = "your api key"
     secretKey = "your secret key"
+    baseURL = "https://www.binance.com" // Can use your own mock environment for testing
```

Comment:
there is no need to move baseURL to NewClient, you can update it simply by `client.BaseURL = xxx`